doc/miniSnip.txt

```diff
@@ -276,7 +276,16 @@ If first (second if description was provided) line starts with `$`, then change
 delimiters in this snippet to strings provided between backtics, e.g.: >
     $ `{{` `}}`
     let foo = {{22}}
-<
+
+-------------------------------------------------------------------------------
+                                                   *'g:miniSnip_expandpattern'*
+Default: '\f'
```

Comment:
We need to think about different default pattern, as per issue #24

Reply:
I suggest \w as default.
\w      word character:                 [0-9A-Za-z_]    /\w
For commonly used characters in filenames without extension.
User could also use custom range [0-9A-Za-z_]